Automattic/crowdsignal-ui
# Add a document section in the editor sidebar

This patch adds a document section to the editor sidebar - for now it's empty.  

Solves c/lhT4SlGu-tr.

I had to add a few CSS tweaks to get the tabs to show up correctly. Normally these styles are included in [@wordpress/edit-post](https://github.com/WordPress/gutenberg/tree/trunk/packages/edit-post) but that package isn't part of [Automattic/isolated-block-editor](https://github.com/Automattic/isolated-block-editor).  
While I was at it, I also fixed other sidebar sections pushing the sidebar too wide and updated the inserter button position to match content width.

# Testing

- Open a project and click on the cog in the right upper corner to trigger the sidebar.
- The sidebar should have two tabs now `Document` and `Block`.
- Verify the other style glitches have been fixed as well.
